snapcraft.yaml

```diff
@@ -169,7 +169,7 @@ parts:
       - -Dxcb=enabled
       - -Dtee=enabled
       - -Dzlib=enabled
-      - -Dgl-backend=disabled
+      #- -Dgl-backend=disabled
```

Comment:
Please remove the line rather than commenting out

Reply:
Done.